secure-api-gateway-ob-uk-rcs-server/src/main/resources/application.yml

```diff
@@ -21,9 +11,65 @@ management:
       exposure:
         include: health, info, prometheus
 
+api:
+  provider:
+    id: 0 # api provider ID (optional)
+    name: Forgerock Bank simulation config # api provider Name (default value: Forgerock Bank Simulation), used to display the account provider in consent user interface
+    address: Forgerock street, 99, XX1 2YY, Forgeshire, uk # api provider address (Optional)
```

Comment:
In that case the client needs to override these values to set his own values, we need the config server to provide a way for the customer to set these values.

Reply:
Yes, this will be tackled in https://github.com/SecureApiGateway/SecureApiGateway/issues/749

For now we want to keep the scope of this PR down to the minimum to achieve removing the config server dependency.

Even without doing the cleanup, customers can override the values of application.yml that is inside our jar, with values in an application.yml found outside of the jar, see: https://docs.spring.io/spring-boot/docs/2.1.9.RELEASE/reference/html/boot-features-external-config.html step 12/13

So plenty of options to play with, I intend to try a few out and make some recommendations in the documentation.

Reply:
Used by ApiProviderConfiguration that is injected in ConsentDetailsApiController to provide the service provider Name displayed in the UI when the consent details is builded to be consumed by the RCS-UI

Reply:
So that we need to move the below section from application-test.yml to application.yml inside of the jar:
```yaml
# General API provider information
# API provider is the organization that exposes the Open Banking API capabilities to be consumed by TPPs (usually a Bank)
api:
  provider:
    id: 0 # api provider ID (optional)
    name: Forgerock Bank simulation config # api provider Name (default value: Forgerock Bank Simulation), used to display the account provider in consent user interface
    address: Forgerock street, 99, XX1 2YY, Forgeshire, uk # api provider address (Optional)
```

Or update the [ApiProviderConfiguration](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-rcs/blob/master/secure-api-gateway-ob-uk-rcs-server/src/main/java/com/forgerock/sapi/gateway/ob/uk/rcs/server/configuration/ApiProviderConfiguration.java) to set a default values for `id` and `address` properties.